third_party/java/java_tools/update_java_tools.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
Sweet! I like scripts because then you need to write less English and there is less room for errors.